src/utils/gitHubUtils.ts

```diff
@@ -103,9 +103,20 @@ export async function getGitHubQuickPicksWithLoadMore<TResult, TParams>(
     }
 }
 
-export async function getGitHubAccessToken(): Promise<string> {
+export async function getGitHubAccessToken(context?: IActionContext): Promise<string> {
     const scopes: string[] = ['repo', 'workflow', 'admin:public_key'];
-    return (await authentication.getSession('github', scopes, { createIfNone: true })).accessToken;
+    try {
+        return (await authentication.getSession('github', scopes, { createIfNone: true })).accessToken;
+    } catch (error) {
+        if (parseError(error).message === 'User did not consent to login.') {
```

Comment:
There's no error code or anything that I can refer to:
![image](https://user-images.githubusercontent.com/5290572/99020471-90dde680-2513-11eb-929f-b55e105f6311.png)

But I don't think this solution will work for other languages since it should be localized...

Reply:
It's not just about other languages - the english text can easily change and break this logic. Ideally you would file an issue on VS Code to give us a better way to detect this error and add a link in the code to that issue. That being said, in this case I don't care that much since the downside isn't that bad (aka they just see the "consent" message)

Reply:
Filed here https://github.com/microsoft/vscode/issues/110758